js-sdk/integrations/vanilla/installation.mdx

```diff
@@ -27,32 +27,123 @@ const { Tolgee } = window['@tolgee/web'];
+import { Tolgee } from 'https://cdn.jsdelivr.net/npm/@tolgee/web@6/dist/tolgee-web.production.esm.min.mjs'
 ```
 
-## In-context (DevTools)
+## Using In-context translation without a bundler
 
-To use In-context in development mode, you can simply use [`DevTools`](../../api/web_package/plugins#devtools) from `@tolgee/web`
+### For Development
 
-```ts
-import { Tolgee, DevTools } from '@tolgee/web';
+For development, you can use the `.development.` bundle, which includes in-context tools. This allows you to translate your app in-context.
+
+```html
+<script src="https://cdn.jsdelivr.net/npm/@tolgee/web@6/dist/tolgee-web.development.umd.js"></script>
+<script>
+  const { Tolgee, DevTools } = window['@tolgee/web'];
 
-const tolgee = Tolgee().use(DevTools()).init(...)
+  const tolgee = Tolgee()
+    .use(DevTools())
+    .init({
+      // ...
+    });
+</script>
 ```
 
-If you use any standard FE setup with a bundler, `DevTools` will be automatically excluded in production (based on `process.env.NODE_ENV`). If you want the in-context to be always available, use [`InContextTools`](/api/web_package/tools.mdx#incontexttools) instead of `DevTools`- it is the same plugin, the only difference is that `DevTools` are exported conditionally.
+### For Production
 
-## Using dev tools without bundler
+For production, use the `.production.` bundle. It's smaller as it doesn't contain the in-context translating functionality.
 
-You can use separately bundled `in-context-tools` however, you'll have to make sure you won't include it in production.
+```html
+<script src="https://cdn.jsdelivr.net/npm/@tolgee/web@6/dist/tolgee-web.production.umd.min.js"></script>
+<script>
+  const { Tolgee } = window['@tolgee/web'];
 
-```jsx
-<script src="https://cdn.jsdelivr.net/npm/@tolgee/web/dist/tolgee-in-context-tools.umd.min.js"></script>;
-const { InContextTools } = window['@tolgee/in-context-tools'];
+  const tolgee = Tolgee()
+    .init({
+      // ...
+    });
+</script>
+```
+
+### Environment Detection Example
+
+You can use environment detection to automatically switch between bundles:
+
+```html
+<!DOCTYPE html>
+<html>
+<head>
+  <title>My App with Tolgee</title>
+</head>
+<body>
+  <div id="app">
+    <h1 data-tolgee-key="title">Welcome</h1>
+  </div>
+
+  <script>
+    // Detect if running locally or in development
+    const isDevelopment = window.location.hostname === 'localhost' || 
+                         window.location.hostname === '127.0.0.1' ||
+                         window.location.hostname.includes('vercel.app');
+
+    const scriptUrl = isDevelopment
+      ? 'https://cdn.jsdelivr.net/npm/@tolgee/web@6/dist/tolgee-web.development.umd.js'
+      : 'https://cdn.jsdelivr.net/npm/@tolgee/web@6/dist/tolgee-web.production.umd.min.js';
+
+    // Load the appropriate script
+    const script = document.createElement('script');
+    script.src = scriptUrl;
+    script.onload = initTolgee;
+    document.head.appendChild(script);
```

Comment:
Aren't we able to do this with the es import? This seems kinda hacky, simple `await import(...)` would look more modern and clean